src/components/header/examples/external-hero/index.njk

```diff
@@ -5,7 +5,7 @@
 
 {{
     onsHeader({
-        "classes": 'ons-header--hero',
+        "variants": 'hero',
```

Comment:
It was doing more but based on if a description was added. I've removed all that to make the code easier to understand. The flex added to the parent grid column handles the flex layout. What I did notice is that flex wasn't being used for the title + description because the description element was outside of all that. So i've moved it inside so the button can vertically align correctly.